packit.yaml

```diff
@@ -28,9 +28,10 @@ jobs:
     trigger: pull_request
     targets:
       - fedora-40
-      - fedora-41
```

Comment:
Why do we need to remove this here? I thought it will just be ignored by TF until they have an image? Or are you afraid that once the image appears, it will be red out of the blue?

Reply:
Ah, I see, it's an actual error: https://github.com/cockpit-project/cockpit/pull/20916/checks?check_run_id=28964505167

Reply:
Yes, our PRs started to be all red due to this; it's a COPR target, and *packit* knows it, but TF doesn't have an image yet. That's a bit awkward.